.github/workflows/build.yml

```diff
@@ -87,7 +87,7 @@ jobs:
           -DBUILD_DEV_PLUGINS:BOOL=${{ matrix.plugins == 'all' }} \
           -DBUILD_SIZECHECK:BOOL=${{ matrix.plugins == 'all' }} \
           -DBUILD_SKELETON:BOOL=${{ matrix.plugins == 'all' }} \
-          -DBUILD_STONESENSE:BOOL=${{ matrix.plugins == 'all' }} \
+          -DBUILD_STONESENSE:BOOL=1 \
```

Comment:
this is the for the linux build actions. You'd need to pass `-DBUILD_STONESENSE:BOOL=1` through `build-win64-from-linux.sh` on line 148

Reply:
which may take some editing of `build-win64-from-linux.sh`

Reply:
ohhhhh i see. Thanks

Reply:
yeah, it looks like the build script isn't flexible enough to allow extra build args at the moment. Sorry I didn't check it before sending you down this path :/

Reply:
No problem at all. Im happy to learn alil anyways.

Reply:
you can either add this directly to the script, right after `-DBUILD_DOCS=1` (most people would want to build stonesense I think?) or add `$EXTRA_CMAKE_ARGS` there, and pass it in line 148 below, e.g.:

```
 env EXTRA_CMAKE_ARGS="-DBUILD_STONESENSE:BOOL=1" bash -x build-win64-from-linux.sh
```

Though will the `EXTRA_CMAKE_ARGS` version we may need to be careful about quoting if we're gonna do multiple `-D`s in the future.

Reply:
Yeah @myk002 recommended the idea of using "$@" so we could just take in as many as we want in the future.

Reply:
The problem is that it needs to be double quoted, as it's being passed into `bash -c`. Or refactored so that it's no longer necessary (we need to run multiple commands inside the docker container).

Reply:
Yeah just testing it, you can just put in $@ (without quotes, since it's already quoted), and it should be fine, but I *think* it can't handle arguments with spaces in them (which should be OK, since I feel like 90% of the shell scripts out there don't get quoting right)

Reply:
I'm not too concerned about the `EXTRA_CMAKE_ARGS` approach. The quoting problem I'm expecting, which would happen both with `$@` and an environment variable like `EXTRA_CMAKE_ARGS`, is preserving spaces within individual arguments, which we should pretty much never need to do (e.g. there is no space in `-DBUILD_STONESENSE:BOOL=1`). `EXTRA_CMAKE_ARGS` would support multiple arguments separated by spaces without a problem.